packages/theme/src/style.module.css

```diff
@@ -0,0 +1,3 @@
+.root {
+	display: contents;
+}
```

Comment:
Removed `@layer`s, since I don't think anybody will need to override these (97aa62159a20c8e8aae974b96bc5377495d9e1a2).